xbmc/guilib/GUIFontCache.cpp

```diff
@@ -32,6 +32,10 @@ class CGUIFontCacheImpl
     using HashIter = typename HashMap::iterator;
     using AgeMap = std::multimap<size_t, HashIter>;
 
+    ~EntryList()
+    {
+      Flush();
+    }
```

Comment:
While obviously right. Do we have a code exit that does not call Flush() manually?